b827ebfffe17de87.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "b827ebfffe17de87", 
```

Comment:
Please use uppercase for the ID here and also in the file name.

Reply:
Thank you for a reply, but I just change Global_conf.json and local_conf.json already by my self and thank you again.